Run workflow updates for saved cloud plans (#433)
hashicorp/terraform-docs-common: website/docs/cloud-docs/run/remote-operations.mdx

Whenever a new run is initiated, it's added to the end of the queue. If there's already a run in progress, the new run won't start until the current one has completely finished — Terraform Cloud won't even plan the run yet, because the current run might change what a future run would do. Runs that are waiting for other runs to finish are in a _pending_ state, and a workspace might have any number of pending runs.

There are two exceptions to the run queue, which can proceed at any time and do not block the progress of other runs:

- Plan-only runs.
- The planning stages of [saved plan runs](/terraform/cloud-docs/run/modes-and-options.mdx#saved-plans). (Note that you can only _apply_ a saved plan if no other run is in progress, and the apply will block the run queue like normal.)

When you initiate a run, Terraform Cloud locks the run to a particular configuration version and set of variable values. If you change variables or commit new code before the run finishes, it will only affect future runs, not runs that are already pending, planning, or awaiting apply.

### Workspace Locks

> When a workspace is _locked,_ new runs can be created (automatically or manually) but will not begin until the workspace is unlocked.

Author: let's keep the "can" tho

```suggestion
When a workspace is _locked,_ Terraform Cloud can create new runs (automatically or manually), but those runs do not begin until you unlock the workspace.
```
